src/Microsoft.OData.Client/PublicAPI/net48/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,1030 @@
+Microsoft.OData.Client.UriFunctionAttribute

```

Comment:
How come your comment here https://github.com/OData/odata.net/pull/2851/files#r1482628586, doesn't apply to this Shipped.txt file?

Reply:
You and @KenitoInc are right. I had not noticed that some are in shipped and some in unshipped. I'll correct this.

Reply:
Fixed. Moved all the API declarations to `Unshipped.text` files.